packages/adapter-static/index.js

```diff
@@ -11,6 +11,8 @@ export default function ({ pages = 'build', assets = pages, fallback } = {}) {
 		name: '@sveltejs/adapter-static',
 
 		async adapt({ utils }) {
+			if (assets !== pages) utils.rimraf(assets);
```

Comment:
I read this line and it didn't make any sense to me. I was sitting there puzzling over it for a few minutes before I read the next line and understood. It might make sense to reverse the order of the two lines. Or maybe better yet we don't even need the `if` since I'd expect it to have no negative effect or have much performance impact if we run it twice